wisdem/inputs/geometry_schema.yaml

```diff
@@ -2733,10 +2775,10 @@ properties:
                 default: 2.
             air_speed_sound:
```

Comment:
Can we rename this to be just speed_sound?

Reply:
I did this in the past, but abandoned it because input names can be "sticky."  Users (we) probably have dozens of inputs around (not necessarily version controlled or in this repo) that will no longer work if we change this input name.  I think it's best to maintain some kind of backwards compatibility, and perhaps add a new `speed_sound` input, but I'd prefer to leave that decision to the maintainers.

Reply:
This would be something to take up with WindIO [definitions](https://windio.readthedocs.io/en/latest/source/environment.html)?.  Maybe we should just add `water_speed_sound`?